bika/lims/catalog.py

```diff
@@ -62,6 +62,9 @@
             'getPrinted': 'FieldIndex',
             'getClientSampleID': 'FieldIndex',
             'getSampleID': 'FieldIndex',
+            # allowedRolesAndUsers is obligatory if we are going to run
+            # advancedqueries in this catalog.
+            'allowedRolesAndUsers': 'KeywordIndex',
```

Comment:
@Espurna I think would be better if we move those "base" indexes used by Plone to a "generic" catalog definition. The idea is the system to automatically add them in new catalogs, also if not defined explicitely. I was thinking that maybe 'UID', 'portal_type', 'created', 'id', 'getId', 'path', 'sortable_title' and 'portal_type' (and their counterpart columns) could be treated as "defaults" too.